modules/ROOT/pages/shared-tenancy.adoc

```diff
@@ -0,0 +1,90 @@
+= Configuring Shared Tenancy in Runtime Fabric
+
+Shared tenancy enables you to deploy Runtime Fabric alongside other services in a Kubernetes cluster. By limiting permissions in your Runtime Fabric cluster, you can allow access only to resources within namespaces assigned to Runtime Fabric. 
+
+With shared tenancy, you have control of the creation of the following Runtime Fabric resources:
+
+* Namespaces
+* RoleBindings
+* ServiceAccounts
+
+== How Shared Tenancy Works
+
+At install time, you provide a configuration file that consists of your namespaces, ServiceAccounts, ClusterRoles/Roles, and RoleBindings.
+
+Runtime Fabric installs the rtf-agent in the Runtime Fabric namespace, and it has access to any additional namespaces you create.
+
+=== Example
+
+The following is an example of a rtf:agent ClusterRole that allows everything regarding cronjobs. available in the whole cluster, but doesn’t itself grant permissions for the whole cluster. It’s available in the whole cluster, but doesn’t itself grant permissions for the whole cluster.
```

Comment:
This paragraph is confusing and repeats text. What I'm understanding from this is:

The following is an example of a _rtf:agent ClusterRole_ that allows every action/command regarding _cronjobs_ that are available in the cluster but does not grant any additional permissions for the cluster.

Please check if this is correct and make the necessary modifications.

Reply:
ooof, copy/paste error, but you're right.

Reply:
This is what it's supposed to say 

The following is an example of a Runtime Fabric agent ClusterRole that allows all cronjob actions (`verbs`). The ClusterRole is available in the whole cluster, but it doesn’t yet have permissions for the whole cluster.